helm/idol-nifi/templates/nifi/service.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nifi
+  labels: {{- include "idol-nifi.labels" . | nindent 4 -}}
+    app: nifi
+  annotations:
+    app.kubernetes.io/name: nifi
+    app.kubernetes.io/part-of: nifi
+spec:
+  type: ClusterIP
+  selector:
+    app: nifi
+  ports:
```

Comment:
Is NiFi going to expose an ACI port (for Universal Viewing)?

Reply:
[e4dbdbe](https://github.com/opentext-idol/idol-containers-toolkit/pull/12/commits/e4dbdbe5ac4d8d45fbb1e33020f6c9e5f5d034be)